.clj-kondo/config.edn

```diff
@@ -403,7 +403,6 @@
     metabase.api.permission-graph                                 api.permission-graph
     metabase.api.permissions                                      api.permissions
     metabase.api.permissions-test-util                            perm-test-util
-    metabase.api.pivots                                           api.pivots
```

Comment:
`metabase.api.pivots` is actually a test util namespace so I renamed it to `metabase.api.pivot-test-util` to make it less confusing.